addons/point_of_sale/models/pos_config.py

```diff
@@ -214,6 +214,8 @@ def _compute_customer_html(self):
     company_has_template = fields.Boolean(string="Company has chart of accounts", compute="_compute_company_has_template")
     current_user_id = fields.Many2one('res.users', string='Current Session Responsible', compute='_compute_current_session_user')
     other_devices = fields.Boolean(string="Other Devices", help="Connect devices to your PoS without an IoT Box.")
+    rounding_method = fields.Many2one('account.cash.rounding', string="Cash rounding")
```

Comment:
Note that the JS only does a half-up rounding, so the `rounding_method` and `strategy` fields won't have any effect. To be implemented?

Reply:
@rhe-odoo Checlk what nim said here, you can round down with 'floor' and round up with 'ceil' depending on how it should be rounded